cedar-policy-validator/src/cedar_schema/to_json_schema.rs

```diff
@@ -162,6 +162,12 @@ fn convert_namespace(
     Ok((ns_name, def))
 }
 
+// Test if this id is a reserved JSON schema keyword.
+// Issue: https://github.com/cedar-policy/cedar/issues/1070
+fn is_reserved_json_schema_keyword(id: &UnreservedId) -> bool {
```

Comment:
Here as well, these are reserved in both syntaxes right? so we can just say "reserved schema keyword"

Reply:
Done.